aten/src/ATen/native/sparse/SparseMatMul.cpp

```diff
@@ -0,0 +1,231 @@
+#include <ATen/ATen.h>
+#include <ATen/Config.h>
+#include <ATen/NamedTensorUtils.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/Parallel.h>
+#include <ATen/SparseTensorImpl.h>
+#include <ATen/SparseTensorUtils.h>
+
+namespace at {
+namespace native {
+
+// helper
+namespace {
+
+template <typename scalar_t>
+std::vector<int64_t> sort_indices(const TensorAccessor<scalar_t, 1>& v) {
```

Comment:
Is this function actually used? I couldn't find a callsite for it.

Reply:
Thanks, dead code removed